Assets/Scripts/CameraControl.cs

```diff
@@ -38,7 +41,7 @@ void Update() {
             Vector3 offset = rotation * cameraDistance;
             transform.position = target.position + offset;
             
-            target.rotation = Quaternion.Euler(0, yaw, 0);
+            target.rotation = targetStartRotation * Quaternion.Euler(0, yaw, 0);
```

Comment:
@XanderConway I feel that it's inappropriate to put code concerning the starting angle in the update method. I think initializing the angle in the start method is a better separation of responsibilities, my suggestion is:

```
void Start() {
        controls = target.GetComponentInChildren<PogoControls>();
        Vector3 initialRotation = transform.eulerAngles - target.eulerAngles;
        yaw = initialRotation.y;
        pitch = initialRotation.z;
}
```